spec/models/hydrus/generic_object_spec.rb

```diff
@@ -267,53 +267,6 @@ def genre
   end
 
   describe 'workflow stuff' do
-    before(:each) do
```

Comment:
The code this was testing wasn't used anywhere in the application, so I excised it.